tools/ThermoRawFileParser/thermo_converter.xml

```diff
@@ -1,145 +1,202 @@
-<tool id="thermo_raw_file_converter" name="Thermo" version="1.2.3+galaxy0">
+<tool id="thermo_raw_file_converter" name="Thermo" version="@TOOL_VERSION@+galaxy0" profile="21.01">
```

Comment:
Can we drop the profile a bit down then? Is there a required target?

Reply:
:+1:

Reply:
> Can we drop the profile a bit down then?

Sure. I went for 20.05

> Is there a required target?

Not really.